### PR TITLE
Support reactions on Voice Broadcast

### PR DESCRIPTION
**Pull request by yostyle**
#7807

### CODE REVIEW

File: vector/src/main/java/im/vector/app/core/extensions/TimelineEvent.kt [hunk 1]
```diff
@@ -27,7 +27,8 @@ import org.matrix.android.sdk.api.session.room.timeline.getLastMessageContent
 
 fun TimelineEvent.canReact(): Boolean {
     // Only event of type EventType.MESSAGE, EventType.STICKER and EventType.POLL_START are supported for the moment
-    return root.getClearType() in listOf(EventType.MESSAGE, EventType.STICKER) + EventType.POLL_START.values + EventType.POLL_END.values &&
+    return root.getClearType() in listOf(EventType.MESSAGE, EventType.STICKER, VoiceBroadcastConstants.STATE_ROOM_VOICE_BROADCAST_INFO) +
```
Comment (Florian14) [inline vector/src/main/java/im/vector/app/core/extensions/TimelineEvent.kt:30]: we should add a check on the event content to only allow reactions on the started event (the stopped is also displayed in the timeline)

Comment (yostyle) [reply vector/src/main/java/im/vector/app/core/extensions/TimelineEvent.kt]: I added this condition

File: vector/src/main/java/im/vector/app/core/extensions/TimelineEvent.kt [hunk 2]
```diff
@@ -26,9 +28,10 @@ import org.matrix.android.sdk.api.session.room.timeline.TimelineEvent
 import org.matrix.android.sdk.api.session.room.timeline.getLastMessageContent
 
 fun TimelineEvent.canReact(): Boolean {
-    // Only event of type EventType.MESSAGE, EventType.STICKER and EventType.POLL_START are supported for the moment
-    return root.getClearType() in listOf(EventType.MESSAGE, EventType.STICKER, VoiceBroadcastConstants.STATE_ROOM_VOICE_BROADCAST_INFO) +
-            EventType.POLL_START.values + EventType.POLL_END.values &&
+    // Only event of type EventType.MESSAGE, EventType.STICKER and EventType.POLL_START, and started voice broadcast are supported for the moment
+    return (root.getClearType() in listOf(EventType.MESSAGE, EventType.STICKER) + EventType.POLL_START.values + EventType.POLL_END.values ||
+            root.getClearType() == VoiceBroadcastConstants.STATE_ROOM_VOICE_BROADCAST_INFO &&
```
Comment (Florian14) [inline vector/src/main/java/im/vector/app/core/extensions/TimelineEvent.kt:33]: You don't need anymore, if the event is not a VB, `root.asVoiceBroadcastEvent()` will return `null`.
Also, do you think we should add parenthesis?

**Review by Florian14 — APPROVED** [review]
LGTM, thanks

**Pull request merged**